memo-app/src/App.js

```diff
@@ -0,0 +1,92 @@
+import { useState, useEffect } from "react";
+import { v4 as uuidv4 } from "uuid";
+import "./App.css";
+import MemoList from "./MemoList.js";
+import MemoContent from "./MemoContent.js";
+import AddButton from "./AddButton.js";
+
+function App() {
+  const registeredMemo =
+    localStorage.getItem("Memos") === null
+      ? []
+      : JSON.parse(localStorage.getItem("Memos"));
+
+  const [memos, setMemos] = useState(registeredMemo);
+  const [status, setStatus] = useState("isDisplaying");
+  const [text, setText] = useState("");
+  const [selectedId, setSelectedId] = useState("");
+
+  function handleMemoAdd(text) {
+    const nextMemos = [...memos, { id: uuidv4(), content: text }];
+    setSelectedId(nextMemos[nextMemos.length - 1].id);
+    setText(nextMemos[nextMemos.length - 1].content);
+    setMemos(nextMemos);
+  }
+
+  function handleMemoChange(id, text) {
+    const nextMemos = memos.map((memo) => {
+      if (memo.id === id) {
+        return {
+          ...memo,
+          content: text,
+        };
+      } else {
+        return memo;
+      }
```

Comment:
else はなくても良さそうですね ☺️ 

```suggestion
    if (memo.id === id) {
      return {
        ...memo,
        content: text,
      };
    }
    return memo;
```

Reply:
ありがとうございます！
ご指摘のとおり不要でしたので、elseは削除いたしました🙏